project/ChatBuild.scala

```diff
@@ -44,8 +45,20 @@ object ChatBuild extends Build {
             .map((f1, f2) => Seq(f1.data, f2.data)),
         watchSources <++= (watchSources in frontend)
       )
+      .dependsOn(sharedJvm)
+
+  lazy val shared = (crossProject.crossType(CrossType.Pure) in file ("shared")).
+    settings(
+      scalaVersion:=scalaV,
+      libraryDependencies +="org.scala-js" %% "scalajs-stubs" % scalaJSVersion % "provided"
```

Comment:
Ok, no worries. I'll try if it works even without it and delete it in the merge commit.